<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Rania</class>
 <widget class="QDialog" name="Rania">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1226</width>
    <height>680</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="logout">
   <property name="geometry">
    <rect>
     <x>1140</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>logout</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 96, 99);</string>
   </property>
   <property name="text">
    <string>nom_benevol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_6">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 96, 99);</string>
   </property>
   <property name="text">
    <string>id_benevol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_444">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>30</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="TRITRI">
   <property name="geometry">
    <rect>
     <x>880</x>
     <y>30</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(249, 183, 255);</string>
   </property>
   <property name="text">
    <string>TRIER</string>
   </property>
  </widget>
  <widget class="QPushButton" name="rechrech">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>30</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(249, 183, 255);</string>
   </property>
   <property name="text">
    <string>RECHERCHER</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_99">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>290</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 96, 99);</string>
   </property>
   <property name="text">
    <string>id_de_membre</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_101">
   <property name="geometry">
    <rect>
     <x>940</x>
     <y>290</y>
     <width>131</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 96, 99);</string>
   </property>
   <property name="text">
    <string>Nbr_participations</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="UPLOAD">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>140</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(249, 183, 255);</string>
   </property>
   <property name="text">
    <string>UPLOAD VIDEO</string>
   </property>
  </widget>
  <widget class="QPushButton" name="IMPRIMER">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>220</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(249, 183, 255);</string>
   </property>
   <property name="text">
    <string>IMPRIMER PDF</string>
   </property>
  </widget>
  <widget class="QPushButton" name="MEMBRE">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>280</y>
     <width>451</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(249, 183, 255);</string>
   </property>
   <property name="text">
    <string>Membre du mois </string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_Ajoutter">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>260</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(142, 142, 142);</string>
   </property>
   <property name="text">
    <string>Ajouter</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>60</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="nbrpart">
   <property name="geometry">
    <rect>
     <x>1120</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>220</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 96, 99);</string>
   </property>
   <property name="text">
    <string>adresse_benevol</string>
   </property>
  </widget>
  <widget class="QTableView" name="tablebenevol">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>80</y>
     <width>721</width>
     <height>192</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="focusPolicy">
    <enum>Qt::StrongFocus</enum>
   </property>
  </widget>
  <widget class="QLabel" name="id">
   <property name="geometry">
    <rect>
     <x>860</x>
     <y>290</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>180</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 96, 99);</string>
   </property>
   <property name="text">
    <string>tel_benevol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>113</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>531</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 18pt &quot;Courier New&quot;;

color: rgb(243, 72, 84);</string>
   </property>
   <property name="text">
    <string>Gestion  des BENEVOLS</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>140</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 96, 99);</string>
   </property>
   <property name="text">
    <string>prenom_benevol</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_8">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>90</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_9">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>113</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>141</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 96, 99);</string>
   </property>
   <property name="text">
    <string>nom_benevol</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Modiffier">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(142, 142, 142);</string>
   </property>
   <property name="text">
    <string>modifier</string>
   </property>
  </widget>
  <widget class="QPushButton" name="suppprimer">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>350</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(142, 142, 142);</string>
   </property>
   <property name="text">
    <string>Supprimer</string>
   </property>
  </widget>
  <widget class="QPushButton" name="stattt">
   <property name="geometry">
    <rect>
     <x>1010</x>
     <y>70</y>
     <width>161</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-color: rgb(249, 183, 255);</string>
   </property>
   <property name="text">
    <string>Statistiques</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>824</x>
     <y>390</y>
     <width>121</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Verifier Parking</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>816</x>
     <y>459</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
